internal/aws/cfn/common.go

```diff
@@ -17,3 +17,18 @@ func makeTags(tags map[string]string) []types.Tag {
 
 	return out
 }
+
+// uniqueStrings returns a unique subset of the string slice provided.
+func uniqueStrings(input []string) []string {
```

Comment:
We should have unit tests for deterministic helper functions like this.

Reply:
+1